components/autowithdraw-shared.js

```diff
@@ -25,10 +25,15 @@ export function AutowithdrawSettings ({ wallet }) {
     setSendThreshold(Math.max(Math.floor(threshold / 10), 1))
   }, [autoWithdrawThreshold])
 
+  const [mounted, setMounted] = useState(false)
+  useEffect(() => {
+    setMounted(true)
+  }, [])
+
   return (
     <>
       <Checkbox
-        disabled={!wallet.isConfigured}
+        disabled={mounted && !wallet.isConfigured}
```

Comment:
Required to enable the checkbox if a wallet supports both but only payments are configured and we reload the configuration page.

In that case, we render this checkbox on the server as disabled so we need to render it again on the client where the wallet will load the configuration from local storage.